prow/pod-utils/decorate/podspec.go

```diff
@@ -69,6 +69,18 @@ func VolumeMountPaths() []string {
 	return []string{logMountPath, codeMountPath, toolsMountPath, gcsCredentialsMountPath}
 }
 
+// GetLabelsAndAnnotations returns a standard set of labels to add to pod/build/etc resources.
+func GetLabelsAndAnnotations(pj kube.ProwJob) (map[string]string, map[string]string) {
```

Comment:
Amazingly we also have half of this logic in `pjutil` -- can we merge that?

Reply:
Also, I know this isn't changing in this PR, but it would be cool to move the label & annotation key/value validation here as well, so we can't fail a POST to the API by using the output of this func

Reply:
Done